cleaners/amule.xml

```diff
@@ -18,42 +18,54 @@
-    <action command="delete" search="file" path="~/.aMule/logfile"/>
-    <action command="delete" search="file" path="~/.aMule/logfile.bak"/>
-    <action command="delete" search="file" path="$APPDATA\aMule\logfile"/>
-    <action command="delete" search="file" path="$APPDATA\aMule\logfile.bak"/>
+    <action command="delete" search="file" path="$$profile$$\logfile"/>
+    <action command="delete" search="file" path="$$profile$$\logfile.bak"/>
   </option>
-  <option id="tmp">
+  <option id="temp">
     <label>Temporary files</label>
     <description>Delete the temporary files</description>
-    <action command="delete" search="walk.files" path="~/.aMule/Temp/"/>
-    <action command="delete" search="walk.files" path="$APPDATA\aMule\Temp\"/>
+    <action command="delete" search="walk.files" path="$$profile$$\Temp\"/>
   </option>
   <option id="known_files">
     <label>Known files history</label>
-    <description>Delete history of known and cancelled files</description>
-    <action command="delete" search="file" path="~/.aMule/known.met"/>
-    <action command="delete" search="file" path="~/.aMule/known2_64.met"/>
-    <action command="delete" search="file" path="~/.aMule/canceled.met"/>
-    <action command="delete" search="file" path="$APPDATA\aMule\known.met"/>
-    <action command="delete" search="file" path="$APPDATA\aMule\known2_64.met"/>
-    <action command="delete" search="file" path="$APPDATA\aMule\canceled.met"/>
+    <description>Delete the usage history</description>
+    <action command="delete" search="file" path="$$profile$$\known.met"/>
+    <action command="delete" search="file" path="$$profile$$\known2_64.met"/>
+    <action command="delete" search="file" path="$$profile$$\canceled.met"/>
   </option>
   <option id="known_clients">
-    <label>Known clients</label>
-    <description>Delete known clients and friends</description>
-    <action command="delete" search="file" path="~/.aMule/emfriends.met"/>
-    <action command="delete" search="file" path="~/.aMule/clients.met"/>
-    <action command="delete" search="file" path="~/.aMule/clients.met.bak"/>
-    <action command="delete" search="file" path="$APPDATA\aMule\emfriends.met"/>
-    <action command="delete" search="file" path="$APPDATA\aMule\clients.met"/>
-    <action command="delete" search="file" path="$APPDATA\aMule\clients.met.bak"/>
+    <label>Known clients and friends</label>
+    <description>Delete the usage history</description>
+    <action command="delete" search="file" path="$$profile$$\emfriends.met"/>
+    <action command="delete" search="file" path="$$profile$$\clients.met"/>
+    <action command="delete" search="file" path="$$profile$$\clients.met.bak"/>
+  </option>
+  <option id="backup">
+    <label>Backup files</label>
+    <description>Delete the backup files</description>
+    <action command="delete" search="file" path="$$profile$$\logfile.bak"/>
+    <action command="delete" search="file" path="$$profile$$\clients.met.bak"/>
```

Comment:
Both files in the `backup` option are also listed in other places, and this inflates the size of the preview when previewing all the options.